linkify.js

```diff
@@ -15,10 +15,27 @@ function redirectGoLink(details) {
     return {redirectUrl: local_storage_cache[golink].url}
 }
 
+function redirectSearchLink(details) {
+    let golink = details.url.replace(/(.*)\:\/\/(.*)=go%2F/, "")
+    const regex = /%2F/g;
+    golink =  golink.replace(regex, "/");
+    golink = golink.replace(/\&fr=opensearch|\&addon=opensearch/, "")
```

Comment:
what is this?

Reply:
It's to remove the trailing query strings in the search links of Yahoo and Ecosia search. I've added a comment explaining it.